cmd/createtree/main_test.go

```diff
@@ -27,173 +27,132 @@ import (
 	"github.com/golang/protobuf/ptypes/any"
 	"github.com/golang/protobuf/ptypes/empty"
 	"github.com/google/trillian"
-	"github.com/google/trillian/crypto/keys"
 	"github.com/google/trillian/crypto/keyspb"
 	"github.com/google/trillian/crypto/sigpb"
+	"github.com/google/trillian/util/flagsaver"
 	"github.com/kylelemons/godebug/pretty"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 )
 
-func marshalAny(p proto.Message) *any.Any {
+// defaultTree reflects all flag defaults with the addition of a valid private key.
+var defaultTree = &trillian.Tree{
+	TreeState:          trillian.TreeState_ACTIVE,
+	TreeType:           trillian.TreeType_LOG,
+	HashStrategy:       trillian.HashStrategy_RFC6962_SHA256,
+	HashAlgorithm:      sigpb.DigitallySigned_SHA256,
+	SignatureAlgorithm: sigpb.DigitallySigned_RSA,
```

Comment:
I agree that ECDSA should be the default, but is it reasonable to change the default given that this flag has existed for a while? It could surprise some people if they're currently assuming that they'll get RSA trees if they don't provide this flag.